src/lib.rs

```diff
@@ -142,7 +142,7 @@
 //!
 //! ```no_run
 //! use std::io::{stdout, Write};
-//! use crossterm::{execute, queue, Color, PrintStyledFont, Colorize, Goto, Clear, ClearType, Result};
+//! use crossterm::{execute, queue, Color, PrintStyledFont, Colorize, cursor::MoveTo, Clear, ClearType, Result};
```

Comment:
Long line. Horizontal scrollbar in the documentation, because it shows just:

```
use crossterm::{execute, queue, Color, PrintStyledFont, Colorize, cursor::MoveTo, Clear, ClearTy
```